src/cli/utils/versions.ts

```diff
@@ -1,19 +1,17 @@
 import type SemVer from 'semver/classes/semver';
 import semverCoerce from 'semver/functions/coerce';
 import semverGte from 'semver/functions/gte';
-import semverParse from 'semver/functions/parse';
 import semverSatisfies from 'semver/functions/satisfies';
 import semverSort from 'semver/functions/sort';
-import semverValid from 'semver/functions/valid';
 
 export { semverGte, semverSort, semverCoerce, semverSatisfies };
 
 export function isValid(version: string): boolean {
-  return semverValid(version) !== null;
+  return semverCoerce(version, { loose: true }) !== null;
```

Comment:
no, this causes unwanted side effects to other tools. only do it in tool installer class

Reply:
ack